.travis.yml

```diff
@@ -1,10 +1,6 @@
 sudo: false
-addons:
-  apt:
-    packages:
-    - oracle-java8-installer
 language: scala
 jdk:
-- oraclejdk8
+- openjdk8
```

Comment:
Switched to openjdk8 since oraclejdk8 isn’t supported on Xenial. As suggested in https://travis-ci.community/t/install-of-oracle-jdk8-is-failing/4365/2